packages/paypal-commerce-integration/src/paypal-commerce-accelerated-checkout/paypal-commerce-accelerated-checkout-customer-strategy.spec.ts

```diff
@@ -147,6 +147,7 @@ describe('PayPalCommerceAcceleratedCheckoutCustomerStrategy', () => {
 
         const state = paymentIntegrationService.getState();
 
+        // jest.spyOn(paymentIntegrationService, 'getState');
```

Comment:
Do we need this line?

Reply:
Nope, but this is revert PR with no changes. I will remove the line in the future